### PR TITLE
add new track api url

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1179

## Description
Fetch track categories using new track api

## Deployment URL
http://use-track-api.review.ensembl.org/genome-browser/homo_sapiens_GCA_000001405_28?focus=gene:ENSG00000139618&location=13:32273076-32442662

## Views affected
GB

### Other effects
NA

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
NA

### CODE REVIEW

File: src/ensembl/src/shared/state/genome/genomeActions.ts [hunk 1]
```diff
@@ -108,10 +101,11 @@ export const fetchGenomeTrackCategories: ActionCreator<ThunkAction<
 
     dispatch(fetchGenomeTrackCategoriesAsyncActions.request(genomeId));
 
-    const url = `/api/genomesearch/genome/track_categories?genome_id=${genomeId}`;
+    const url = `/api/tracks/track_categories/${genomeId}`;
     const response = await apiService.fetch(url);
-    updatedGenomeTrackCategories[genomeId] = response.track_categories;
 
+    updatedGenomeTrackCategories[genomeId] =
+      response.track_categories.reverse();
```
Comment (azangru) [inline src/ensembl/src/shared/state/genome/genomeActions.ts:108]: Let's talk to the backend team about this.

**Comment by imransl** [comment]
@jyothishnt @azangru We still seem to be having this issue with Prettier. Makes it quite difficult to review the PR 😞

File: src/ensembl/src/shared/state/genome/genomeActions.ts (hunk 1 above)
Comment (jyothishnt) [reply src/ensembl/src/shared/state/genome/genomeActions.ts:108]: @veidenberg will update the track api to return the correct order of categories

Comment (veidenberg) [reply src/ensembl/src/shared/state/genome/genomeActions.ts:108]: I have updated the category order in the backend side ([example request](http://staging-2020.ensembl.org/api/tracks/track_categories/homo_sapiens_GCA_000001405_28)).

**Review by azangru — APPROVED** [review]

**Pull request merged**